tests/Test/TestCaseUtils.php

```diff
@@ -38,4 +41,25 @@ public static function swapExpectedInputTestCases(iterable $cases): iterable
             yield $case;
         }
     }
+
+    public static function getFixerByName(string $name): FixerInterface
+    {
+        static $fixers = null;
+
+        if (null === $fixers) {
+            $factory = new FixerFactory();
+            $factory->registerBuiltInFixers();
+
+            $fixers = [];
+            foreach ($factory->getFixers() as $fixer) {
+                $fixers[$fixer->getName()] = $fixer;
+            }
+        }
+
+        if (!\array_key_exists($name, $fixers)) {
```

Comment:
`isset` is more performant I think:
```suggestion
        if (!isset($fixers[$name])) {
```

Reply:
Oops, "nice to have" suggestions does not work nicely with auto-merge 😅.

Reply:
suggestion - ignore this single usage, confirm what is more performant, apply locally with CI check